src/app/data-planes/components/DataPlaneDetails.vue

```diff
@@ -1,81 +1,105 @@
 <template>
-  <TabsWidget :tabs="filteredTabs">
+  <TabsWidget :tabs="TABS">
```

Comment:
Same comment as the previous PR, should this use nested routing? (at the very least in a later PR)